doc/code/qml_tape.rst

```diff
@@ -1,176 +1,26 @@
+gradients.
 
-      @qml.qnode(dev)
-      def circuit():
-          qml.Hadamard(wires=1)
-          return qml.state()
-
-  >>> circuit()
-  array([0.70710678+0.j, 0.70710678+0.j, 0.        +0.j, 0.        +0.j])
-
-  Calculating the derivative of :func:`~.state` is currently only supported when using the
-  classical backpropagation differentiation method (``diff_method="backprop"``) with a
-  compatible device.
-
-* **Less restrictive QNode signatures**: There is no longer any restriction on the QNode signature; the QNode can be
-  defined and called following the same rules as standard Python functions.
-
-  For example, the following QNode uses positional, named, and variable
-  keyword arguments:
-
-  .. code-block:: python
-
-      qml.enable_tape()
-
-      x = torch.tensor(0.1, requires_grad=True)
-      y = torch.tensor([0.2, 0.3], requires_grad=True)
-      z = torch.tensor(0.4, requires_grad=True)
-
-      @qml.qnode(dev, interface="torch")
-      def circuit(p1, p2=y, **kwargs):
-          qml.RX(p1, wires=0)
-          qml.RY(p2[0] * p2[1], wires=0)
-          qml.RX(kwargs["p3"], wires=0)
-          return qml.var(qml.PauliZ(0))
-
-  When we call the QNode, we may pass the arguments by name
-  even if defined positionally; any argument not provided will
-  use the default value.
-
-  >>> res = circuit(p1=x, p3=z)
-  >>> print(res)
-  tensor(0.2327, dtype=torch.float64, grad_fn=<SelectBackward>)
-  >>> res.backward()
-  >>> print(x.grad, y.grad, z.grad)
-  tensor(0.8396) tensor([0.0289, 0.0193]) tensor(0.8387)
-
-* **Unifying all QNodes**: The tape-mode QNode merges all QNodes (including the :class:`~.JacobianQNode`
-  and the :class:`~.PassthruQNode`) into a single unified QNode, with identical behaviour regardless
-  of the differentiation type.
-
-  In addition, it is now possible to inspect the internal variational quantum circuit structure
-  of QNodes when using classical backpropagation (which is not support in the standard
-  :class:`~.PassthruQNode`).
-
-* **Optimizations**: Tape mode provides various performance optimizations, reducing pre- and post-processing
-  overhead, and reduces the number of quantum evaluations in certain cases.
-
-
-Quantum tapes
--------------
-
-Under the hood, tape mode is able to provide these new features by significantly overhauling
-the internal structure of the QNode. When tape mode is enabled, the QNode is no longer
-responsible for recording quantum operations, executing devices, or computing gradients---these
-tasks have been delegated to an internal object that is created by the QNode, the **quantum tape**.
-
-In addition to being created internally by QNodes in tape mode, quantum tapes can also be created,
+In addition to being created internally by QNodes, quantum tapes can also be created,
 nested, expanded (via :meth:`~.QuantumTape.expand`), and executed manually. Tape subclasses also provide
 additional gradient methods:

```

Comment:
I think this is outdated?

Reply:
oops, I think I deleted the autosummary by accident 🤦 
```suggestion
additional gradient methods:

.. autosummary::

    ~pennylane.tape.QuantumTape
    ~pennylane.tape.QubitParamShiftTape
    ~pennylane.tape.CVParamShiftTape
    ~pennylane.tape.ReversibleTape
```